### PR TITLE
PAR-1293: Added restore functionality.

### PR DESCRIPTION
**Pull request by kalpaitch**

### CODE REVIEW

File: web/modules/features/par_rd_help_desk_flows/src/Form/ParRdHelpDeskUnrevokeForm.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+/**
+ * Revoking a partnership.
+ */
+class ParRdHelpDeskUnrevokeForm extends ParBaseForm {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function titleCallback() {
+    return "Help Desk | Partnership restored";
```
Comment (jon-edery) [inline web/modules/features/par_rd_help_desk_flows/src/Form/ParRdHelpDeskUnrevokeForm.php]: restored -> restore ?

Comment (kalpaitch) [reply web/modules/features/par_rd_help_desk_flows/src/Form/ParRdHelpDeskUnrevokeForm.php]: Correct as it is. The two screens go 'Confirm you want to do this' => 'You've done this.'

Comment (jon-edery) [reply web/modules/features/par_rd_help_desk_flows/src/Form/ParRdHelpDeskUnrevokeForm.php]: 👍

**Comment by jon-edery** [comment]
Tests need updating stating the obvious

**Pull request merged**